src/ir_Carrier.cpp

```diff
@@ -182,6 +192,100 @@ bool IRrecv::decodeCarrierAC40(decode_results *results, uint16_t offset,
 }
 #endif  // DECODE_CARRIER_AC40
 
+#if SEND_CARRIER_AC48
+/// Send a CARRIER_AC48 formatted message.
+/// Alternative >64bit function to send CARRIER_AC48 messages
+/// Status: ALPHA / Untested.
+/// Where data is:
+///   uint8_t data[kCARRIER_AC48StateLength] = {0xA1, 0x20, 0x6B, 0xFF, 0xFF, 0xC5, 0x5E, 0xDF, 0x94, 0x00, 0x00, 0x3A};
+///
+/// @param[in] data An array of bytes containing the IR command.
+///                 It is assumed to be in MSB order for this code.
+/// @param[in] nbytes Nr. of bytes of data in the array. (>=kCARRIER_AC48StateLength)
+/// @param[in] repeat Nr. of times the message is to be repeated.
+void IRsend::sendCarrier_AC48(const uint8_t data[], const uint16_t nbytes, const uint16_t repeat) {
```

Comment:
Do we want this to only use 6 bytes (in a uint64_t) and do the inverted part internally?
See `sendCarrierAC` function

Reply:
My typical "guide" is if it fits inside a `uint64_t` then use that, rather than `state[]` approach.
If it is 64 bits exactly, then either way is acceptable if justified.
As the 64 bit carrier protocol is in a `uint64_t` we should follow that for "style" for Carriers.